### DISCUSSION TITLE
Matrix OS 2.5 Release Canidate - Taste of Matrix OS

### DISCUSSION THREAD

**Discussion by 203Null** [203-Systems/MatrixOS#41] [Announcements]
<details><summary>English</summary>

## Taste Of Project Matrix
Added many useful, cool, or random features to the OS. It's a great showcase of what Matrix OS & Mystrix is capable of.

New: System Global Animation (Toggleable)
New: [Companion App](https://matrix.203.io/docs/MatrixOS/Applications/Companion) - [BitFocus Companion)](https://bitfocus.io/companion) Integration 
New: [Game Pad APP](https://matrix.203.io/docs/MatrixOS/Applications/Gamepad) (As a demo to customizable keymap)
New: [Dice APP](https://matrix.203.io/docs/MatrixOS/Applications/Dice)
New: [Reversi Game APP](https://matrix.203.io/docs/MatrixOS/Applications/Reversi)
New: Performance App [Touch Combo Key](https://matrix.203.io/docs/MatrixOS/Applications/Performance#touch-combo-keys)
New: Better LED Subsystem for Mystrix with dithering support
New: [LED Partition - Supports adjusting brightness multiplier for Mystrix Pro Underglow](https://matrix.203.io/docs/MatrixOS/SystemSettings#brightness-control-menu)
New: Dual MIDI port - Allows Apollo Studio to connect with Mystrix without special driver support
New: Underglow for Mystrix boot animation
New: Underglow in Note APP note view.
New: Toggle Button now shows On or Off in their text scroll
New: Mystrix Pro Keypad User Calibration
New: Note APP now can spilled pad vertically
New: [Performance APP now supports 4 customizable palettes.](https://matrix.203.io/docs/MatrixOS/Applications/Performance#color-palette)
Updated: [Lighting App V2](https://matrix.203.io/docs/MatrixOS/Applications/Lighting)
Improved: Color Picker UI
Improved: Better Keypad driver for Mystrix Standard & Mystrix Pro.
Improved: Better LED brightness steps.
Improved: Bluetooth Secure Connection - Enables Windows and Android Support.  (via @M4kI9H7)
Fixed: APPs out of order in App Launcher
Fixed: Fixed some memory leaks
Fixed: Performance App Flicker Reduction Toggle
Fixed: Note App overlap default value not showing correctly
Fixed: Issue sending some special midi packet (#28)
Changed: Mystrix now shows Mystrix / Mystrix Pro in device name
Changed: There's now a Device Setting page in the system setting for device-specific settings.
Removed: Performance App Compatibility Mode - Unipad supports Mystrix now, no need to keep it around.

</details>

<details>
  <summary>中文</summary>

添加了许多有用、炫酷或乱七八糟的功能到操作系统中。主要是展示 Matrix OS 和 魔矩的各种可能性。

新增：系统全局动画（可开关）  
新增：[Companion APP](https://matrix.203.io/docs/MatrixOS/Applications/Companion) - 集成 [BitFocus Companion](https://bitfocus.io/companion)  
新增：[游戏手柄APP](https://matrix.203.io/docs/MatrixOS/Applications/Gamepad)（作为可自定义键位APP的Beta预览）  
新增：[骰子APP](https://matrix.203.io/docs/MatrixOS/Applications/Dice)  
新增：[翻转棋游戏APP](https://matrix.203.io/docs/MatrixOS/Applications/Reversi)  
新增：表演APP [触控组合键](https://matrix.203.io/docs/MatrixOS/Applications/Performance#touch-combo-keys)  
新增：魔矩全新 LED 子系统，支持色彩抖动  
新增：[LED 分区 - 支持调整 魔矩Pro 底灯的亮度倍数](https://matrix.203.io/docs/MatrixOS/SystemSettings#brightness-control-menu)  
新增：双 MIDI 接口 - Apollo Studio 可以在不需要特殊驱动的情况下连接 Mystrix  
新增：Mystrix 启动动画底灯 
新增：记事应用的底光显示  
新增：切换按钮在文字滚动中显示开/关状态  
新增：魔矩Pro 压力感应用户自校准  
新增：记事应用支持垂直分割按键  
新增：[性能应用现在支持 4 种可自定义的调色板](https://matrix.203.io/docs/MatrixOS/Applications/Performance#color-palette)  

更改：[照明APP V2](https://matrix.203.io/docs/MatrixOS/Applications/Lighting)  
更改：色彩选择器界面  
更改：更好的 魔矩和 魔矩Pro 键盘驱动  
更改：改进的 LED 亮度等级  
更改：蓝牙安全连接 - 支持 Windows 和 Android链接（感谢 @M4kI9H7）  

更改：应用启动器中的应用顺序错误  
更改：修复了一些内存泄漏问题  
更改：性能应用中的闪烁减少选项  
更改：记事应用默认重叠值显示不正确的问题  
更改：发送某些特殊 MIDI 数据包的问题（#28）  

更改：魔矩现在在设备名称中显示为 Mystrix / Mystrix Pro  
更改：系统设置中新增“设备设置”页面，用于设备特定设置  

更改：性能应用兼容模式 - Unipad 现已支持魔矩，不再需要此功能
</details>

<details><summary>Supported Devices/支持设备</summary>

- **MatrixOS-V2.5.0-RC-Mystrix.uf2:**
  - 203 Systems - Mystrix & Mystrix Pro

</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/203-Systems/MatrixOS/releases/tag/V2.5-RC'>Matrix OS 2.5 Release Canidate - Taste of Matrix OS</a>.</em>

**Comment by mastwet** [comment]
666
